RedHat-UX/red-hat-design-system
# docs(back-to-top): add missing back to top image

## What I did

1. Added `back-to-top.svg` that is referenced on the Style and Guidelines pages

## Testing Instructions

1. Visit the Style page in the DP.
2. Ensure there are no broken images, especially under the "Theme" heading.
3. Ensure there is not an errant `>` under the last back to top image on that page.
4. ON the Guidelines page in the DP, scroll down to the "'Back to top' text" heading.
5. Make sure there are no broken images there.
6. All good? Scroll back to the top of this PR and approve it ✅ 

## Notes to Reviewers

Got this image from [Marionne's Back to Top docs PR](https://github.com/RedHat-UX/red-hat-design-system/pull/1526/files#diff-b1309f55aed319be8b5554498cc5cdec9990ad8e214d6f78d9f8e48afeae5a87).
